packages/core/src/App/Containers/Modals/trading-assessment-existing-user/trading-assessment-existing-user.jsx

```diff
@@ -132,9 +133,10 @@ const TradingAssessmentExistingUser = observer(() => {
                             setSubSectionIndex={setSubSectionIndex}
                             class_name='trading-assessment--existing-user'
                             should_move_to_next={should_move_to_next}
+                            is_responsive={!isDesktop}
```

Comment:
did we change corresponding component?

Reply:
It disappeared because after merging the changes were already present in master, so no need to updated in corresponding component. its already update done in diff pr